crates/re_ui/src/list_item2/property_content.rs

```diff
@@ -176,35 +175,67 @@ impl ListItemContent for PropertyContent<'_> {
             action_buttons,
         } = *self;
 
-        // We always reserve space for the action button(s), even if there are none.
-        let action_button_rect = egui::Rect::from_center_size(
-            context.rect.right_center() - egui::vec2(ReUi::small_icon_size().x / 2., 0.0),
-            ReUi::small_icon_size() + egui::vec2(1.0, 1.0), // padding is needed for the buttons
-        );
-
-        let content_width =
-            (context.rect.width() - action_button_rect.width() - ReUi::text_to_icon_padding())
-                .at_least(0.0);
-
-        //TODO(ab): adaptable columns
-        let column_width = ((content_width - ReUi::text_to_icon_padding()) / 2.).at_least(0.0);
+        // │                                                                              │
+        // │◀─────────────────────────────background_x_range─────────────────────────────▶│
+        // │                                                                              │
+        // │ ◀───────────state.left_column_width────────────▶│┌──COLUMN_SPACING           │
+        // │                                                  ▼                           │
+        // │                       ◀──────────────CONTENT────┼──────────────────────────▶ │
+        // │ ┌ ─ ─ ─ ─ ┬ ─ ─ ─ ─ ┬ ┬────────┬─┬─────────────┬─┬─────────────┬─┬─────────┐ │
+        // │                       │        │ │             │││             │ │         │ │
+        // │ │         │         │ │        │ │             │ │             │ │         │ │
+        // │   INDENT       ▼      │  ICON  │ │    LABEL    │││    VALUE    │ │   BTN   │ │
+        // │ │         │         │ │        │ │             │ │             │ │         │ │
+        // │                       │        │ │             │││             │ │         │ │
+        // │ └ ─ ─ ─ ─ ┴ ─ ─ ─ ─ ┴ ┴────────┴─┴─────────────┴─┴─────────────┴─┴─────────┘ │
+        // │ ▲                     ▲         ▲               │               ▲            │
+        // │ └──state.left_x       │         └───────────────────────────────┤            │
+        // │                       │                         ▲               │            │
+        // │       content_left_x──┘           mid_point_x───┘     text_to_icon_padding   │
+        // │                                                                              │
+        //
+        // content_indent = content_left_x - state.left_x
+        // left_column_width = content_indent + icon_extra + label_width + COLUMN_SPACING/2
```

Comment:
rm?

Reply:
Actually I meant that as clarifying math based on the schematic but yeah there is a bitrot hazard here so i'll rm